pages/account/manage/event/[[...data]].js

```diff
@@ -72,7 +72,7 @@ export default function ManageEvent({ BASE_URL, event }) {
   }, [isSpeaking]);
 
   const [price, setPrice] = useState(event.price?.startingFrom || 0);
-  const [color, setColor] = useState(event.color || "");
+  const [color, setColor] = useState("");
```

Comment:
The color is not set when editing, I think this needs to be kept how it was before?

Reply:
yes we need to keep it how it was before, then only the color is set when editing the event.